src/agent/src/device.rs

```diff
@@ -87,84 +88,116 @@ fn pcipath_to_sysfs(root_bus_sysfs: &str, pcipath: &pci::Path) -> Result<String>
     Ok(relpath)
 }
 
-#[derive(Debug, Clone)]
-struct DevAddrMatcher {
-    dev_addr: String,
+// FIXME: This matcher is only correct if the guest has at most one
```

Comment:
ehmm, could you please file a bug to track this ?

Reply:
Fair enough.  See #1669.